### PR TITLE
Coding standards improvements and additions

### PR DESCRIPTION
**Pull request by lucianonooijen**

### CODE REVIEW

**Comment by JeroenVanSteijn** [comment]
You know Travis is failing, right?

File: .eslintrc [hunk 1]
```diff
@@ -2,13 +2,22 @@
...
+        "jquery",
         "react",
         "prettier"
     ],
     "rules": {
-        "prettier/prettier": "error"
+        "strict": 1,
+        "eol-last": 1,
+        "no-else-return": 1,
+        "max-len": [1, 150, 2],
```
Comment (JeroenVanSteijn) [inline .eslintrc]: Max length is 150, but in the standards you changed it to 80

File: .eslintrc [hunk 2]
```diff
@@ -2,13 +2,22 @@
...
+        "prefer-arrow-callback": 1,
+        "prefer-template": 2,
+        "indent": [1, 4, { "SwitchCase": 1, "VariableDeclarator": 1 }],
+        "no-loop-func": 0,
+        "no-nested-ternary": 1
     }
```
Comment (JeroenVanSteijn) [inline .eslintrc]: Maybe there is a rule for ending the file with an empty line?

File: README.md [hunk 3]
```diff
@@ -116,14 +121,18 @@ These are the general rules we use for writing software.
...
+* If the used frameworks allow it, write your media queries mobile first, like `@media (min-width: 30rem)`
 * List related items together
 * If a :hover pseudo class is styled, style :focus the same for accessibility
+* Never use IDs, use classes
 * Use [BEM](http://getbem.com/naming/) for naming CSS classes
-* Name classes descriptively 
+* Name classes descriptively
+* Use double quotes, not single quotes
+* Use only one declaration per line, this makes the code more readable
+* Add a space after the colon of a declaration, so not `margin:1rem;` but `margin: 1rem;`
```
Comment (JeroenVanSteijn) [inline README.md]: Add these things to the linter?

File: package.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Bytecode Coding Standards",
+  "name": "bytecode-coding-standards",
   "version": "1.0.0",
```
Comment (JeroenVanSteijn) [inline package.json]: Version 1.1 now?

**Comment by lucianonooijen** [comment]
I know travis is failing, it shouldn't be turned on for this repo as there is no "real" code in here. I will disable it.

**Comment by JeroenVanSteijn** [comment]
I left more comments

File: .eslintrc (hunk 1 above)
Comment (lucianonooijen) [reply .eslintrc]: Good find, I'll change it to 80 here too

File: .eslintrc (hunk 2 above)
Comment (lucianonooijen) [reply .eslintrc]: It is, EOL-Last I believe it's called, but anyway, it's in there

File: README.md (hunk 3 above)
Comment (lucianonooijen) [reply README.md]: That's built into the defaults of the linter, so no need to add it by hand

File: package.json (hunk 4 above)
Comment (lucianonooijen) [reply package.json]: I'll change it

**Pull request merged**